website/content/api-docs/secret/pki.mdx

```diff
@@ -1178,7 +1178,7 @@ path and takes the value `default`.
 :::
 
 - `name` `(string: "")` - Specifies a role. If set, the following parameters
-  from the role will have effect: `ttl`, `max_ttl`, `generate_lease`, `no_store` and `not_before_duration`.
+  from the role will have effect: `ttl`, `max_ttl`, `generate_lease`, `no_store`,  `not_before_duration` and `basic_constraints_valid_for_non_ca`.
```

Comment:
If the above suggestion is taken, will need to reword this one to make it clear it is conditional.

Reply:
Hi @cipherboy ,

Normally, all your comments are now taken into account.